frontend/src/components/dashboard/DashboardCreate/Finish/index.tsx

```diff
@@ -0,0 +1,46 @@
+import Button from '@components/common/Button';
+import CopyToClipboard from '@components/common/CopyToClipboard';
+import { useNavigate, useParams } from 'react-router-dom';
+import { DASHBOARD_POST_URL } from '@constants/constants';
+import ChevronButton from '@components/common/ChevronButton';
+import S from './style';
+
+interface FinishProps {
+  postUrl: string;
+  postId: string;
+}
+
+export default function Finish({ postUrl, postId }: FinishProps) {
+  const navigate = useNavigate();
+  const { dashboardId } = useParams();
+
+  if (!dashboardId) {
+    throw new Error('ROUTE ERROR');
+  }
+
```

Comment:
dashboardId가 undefined일 때를 대비해서 이렇게 처리해주셨네요!
저도 저번 PR에서 NotFound 페이지로 이동되게 했는데, 사실 dashboardId 없이 url을 입력하면 존재하지 않는 라우팅으로 처리되어 Finish 컴포넌트로 들어올 일이 없지요..🙂
만약 dashboardId가 `undefined | string`로 평가되어 이런 처리를 해주신 거라면 상위 레이어에 ErrorBoundary가 없는 현 상황에선 단언으로 에러를 막는 것이 더 나을수도 있겠습니다.

```suggestion
  const { dashboardId } = useParams() as { dashboardId: string };

  if (!dashboardId) {
    throw new Error('ROUTE ERROR');
  }
```

Reply:
맞아용~ 단순하게 undefined 처리만 해준건데, 저게 더 올바르네요!